### ISSUE TITLE
[Issue]: External thumbnails are no longer loaded as of 10.9.5

### ISSUE THREAD

**Issue by UntouchedWagons** [jellyfin/jellyfin#12171]
### Please describe your bug

I download Let’s Play series off of Youtube and sort them to watch through Jellyfin at a later time. I also download the thumbnails. On June 9 I upgraded the Jellyfin server from 10.9.1 to 10.9.6 and thumbnails created after that date were no longer imported:

![08bacb24-e71f-4c55-944e-779a1951126a](https://github.com/jellyfin/jellyfin/assets/160996/cb9e6709-6a07-4196-af4f-05791a449cf3)

After doing some testing on my testing kube cluster I determined that the regression happened sometime after the release of 10.9.4. If I created a pod running Jellyfin 10.9.4 every thumbnail was loaded without fail. If I recreated the pod running Jellyfin 10.9.5 then no thumbnail would be loaded

### Reproduction Steps

1. Run Jellyfin 10.9.5
2. Create a library of videos with all external image fetchers disabled where the videos have images with identical file names

This happens 100% of the time

### Jellyfin Version

10.9.0

### Environment

```markdown
- OS: Debian 12, Talos 1.7.4
- Linux Kernel: 6.1.0-20-amd64, Unknown
- Virtualization: None, Proxmox VMs
- Clients: Firefox
- Browser: Firefox
- FFmpeg Version: 6.0.1 (In Jellyfin 10.9.5)
- Playback Method: NA
- Hardware Acceleration: NA
- GPU Model: NA
- Plugins: NA
- Reverse Proxy: NA
- Base URL: NA
- Networking: NA
- Storage: NFS
```

### Jellyfin logs

```shell
https://gist.github.com/UntouchedWagons/26a8bec2de5e089e0d45104b0dc32fb8
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
Please post your folder and file layout.

**Comment by UntouchedWagons** [comment]
Will this do?

![Y4Y6RVKh1W](https://github.com/jellyfin/jellyfin/assets/160996/fbb18ae6-3d59-4346-a703-71d32358435c)

**Comment by Shadowghost** [comment]
Fixed by #12039
